### DISCUSSION TITLE
Deleting entries sends me to collections

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#383] [General]
I've managed to create a lot of duplicate entries - when I'm going through tasks, each entry I delete then puts me back on the collections page.

I then need to select jtx board again from the 3-bar menu. This then takes me back into tasks ready for my next deletion.

This is very tiresome and not the way I'd expect deletions to work.

Version: 2.03.01-beta01.gplay (203010001)
V2.3.0

**Comment by DickinsonSa** [comment]
Hi Patrick, scratch this for now - everything seems to be working as expected again. Not sure what caused the glitch - maybe my side.

**Comment by patrickunterwegs** [comment]
Hey @DickinsonSa !
Actually I played around a bit to reproduce it 😉 
This problem is the back-stack. When you go from the list to the collections overview and then back to the list (through the menu - not through the back button), then open a single entry and delete it, the app would basically go "back", but for some reason in this constellation the app goes back to the collections overview (instead of the list, like the app tries to go back twice). 

So it is definitely something that should be improved. It might also have come up because of a library update. I'm any case I'll check it! Thanks for the issue report! 👍
